### ISSUE TITLE
runtime-libraries stress-* build definitions are failing in release/6.0

### ISSUE THREAD

**Issue by ericstj** [dotnet/runtime#100944]
## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=639539
Build error leg or test failing: Docker Linux / Build CLR and Libraries
Pull request: N/A

## Error Message

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssueJsonStepByStep.md).

```json
{
  "ErrorMessage": "COPY failed: no source files were specified",
  "ErrorPattern": "",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```

See https://dev.azure.com/dnceng-public/public/_build?definitionId=132&_a=summary&repositoryFilter=68&branchFilter=553%2C553%2C553%2C553%2C553%2C553%2C553%2C553%2C553

Also failing for stress-http
https://dev.azure.com/dnceng-public/public/_build?definitionId=131&_a=summary&repositoryFilter=68&branchFilter=553%2C553%2C553%2C553%2C553%2C553%2C553%2C553%2C553

 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=639539
**Error message validated:** `[COPY failed: no source files were specified`]
**Result validation:** :white_check_mark: Known issue matched with the provided build.
**Validation performed at:** 4/12/2024 1:13:22 AM UTC

### Report
|Build|Definition|Step Name|Console log|Pull Request|
|---|---|---|---|---|
|[710204](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710204)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/710204/logs/13)||
|[709433](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709433)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/709433/logs/9)||
|[708983](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708983)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/708983/logs/9)||
|[707927](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707927)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/707927/logs/9)||
|[706725](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706725)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/706725/logs/16)||
|[705184](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705184)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/705184/logs/9)||
|[703751](https://dev.azure.com/dnceng-public/public/_build/results?buildId=703751)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/703751/logs/16)||
|[702390](https://dev.azure.com/dnceng-public/public/_build/results?buildId=702390)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/702390/logs/9)||
|[701779](https://dev.azure.com/dnceng-public/public/_build/results?buildId=701779)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/701779/logs/9)||
|[701480](https://dev.azure.com/dnceng-public/public/_build/results?buildId=701480)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/701480/logs/9)||
|[700690](https://dev.azure.com/dnceng-public/public/_build/results?buildId=700690)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/700690/logs/9)||
|[699492](https://dev.azure.com/dnceng-public/public/_build/results?buildId=699492)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/699492/logs/9)||
|[698237](https://dev.azure.com/dnceng-public/public/_build/results?buildId=698237)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/698237/logs/9)||
|[697030](https://dev.azure.com/dnceng-public/public/_build/results?buildId=697030)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/697030/logs/16)||
|[695807](https://dev.azure.com/dnceng-public/public/_build/results?buildId=695807)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/695807/logs/9)||
|[695242](https://dev.azure.com/dnceng-public/public/_build/results?buildId=695242)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/695242/logs/9)||
|[694917](https://dev.azure.com/dnceng-public/public/_build/results?buildId=694917)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/694917/logs/16)||
|[693957](https://dev.azure.com/dnceng-public/public/_build/results?buildId=693957)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/693957/logs/9)||
|[692599](https://dev.azure.com/dnceng-public/public/_build/results?buildId=692599)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/692599/logs/9)||
|[691099](https://dev.azure.com/dnceng-public/public/_build/results?buildId=691099)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/691099/logs/9)||
|[689557](https://dev.azure.com/dnceng-public/public/_build/results?buildId=689557)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/689557/logs/9)||
|[688688](https://dev.azure.com/dnceng-public/public/_build/results?buildId=688688)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/688688/logs/9)||
|[688002](https://dev.azure.com/dnceng-public/public/_build/results?buildId=688002)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/688002/logs/9)||
|[687572](https://dev.azure.com/dnceng-public/public/_build/results?buildId=687572)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/687572/logs/9)||
|[686674](https://dev.azure.com/dnceng-public/public/_build/results?buildId=686674)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/686674/logs/7)||
|[685461](https://dev.azure.com/dnceng-public/public/_build/results?buildId=685461)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/685461/logs/9)||
|[684220](https://dev.azure.com/dnceng-public/public/_build/results?buildId=684220)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/684220/logs/9)||
|[681421](https://dev.azure.com/dnceng-public/public/_build/results?buildId=681421)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/681421/logs/9)||
|[680667](https://dev.azure.com/dnceng-public/public/_build/results?buildId=680667)|dotnet/runtime|Build CLR and Libraries|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/680667/logs/9)||
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|1|7|29|

**Comment by ericstj** [comment]
@antonfirsov looks like this might be your area.  Please have a look.

**Comment by antonfirsov** [comment]
@dotnet/ncl I think we should disable nightly stress runs for 6.0. We don't track them in stress reports, and 6.0 will reach EOL soon. Any pushback?

**Comment by wfurt** [comment]
I think that would be fine. I think the risk is small since we do not do active development on 6.0

**Comment by wfurt** [comment]
should this be closed by #103572 @rzikm ?
